crates/compiler/parse/src/blankspace.rs

```diff
@@ -399,6 +399,8 @@ where
                     let is_doc_comment = state.bytes().first() == Some(&b'#')
                         && (state.bytes().get(1) == Some(&b' ')
                             || state.bytes().get(1) == Some(&b'\n')
+                            || (state.bytes().get(1) == Some(&b'\r')
+                                && state.bytes().get(2) == Some(&b'\n'))
```

Comment:
Maybe worth making a helper function for this, given it's used again on 427? (and seems common to check for newline)